app/models/user.rb

```diff
@@ -26,6 +26,10 @@ def disable_merchant_items
     items.each {|item| item.disable}
   end
 
+  def enable_merchant_items
+    items.each {|item| item.enable}
```

Comment:
Can we refactor to `items.update_all(enabled: true)`

This would reduce it to just one ActiveRecord database call instead of one per item